valkyrie/src/main/java/us/ihmc/valkyrie/ValkyrieRobotModel.java

```diff
@@ -142,7 +142,7 @@ public ValkyrieRobotModel(RobotTarget target, boolean headless, String model, Fo
       jointMap = new ValkyrieJointMap();
       contactPointParameters = new ValkyrieContactPointParameters(jointMap, simulationContactPoints);
       sensorInformation = new ValkyrieSensorInformation(target);
-      highLevelControllerParameters = new ValkyrieHighLevelControllerParameters(target == RobotTarget.REAL_ROBOT, jointMap);
+      highLevelControllerParameters = new ValkyrieHighLevelControllerParameters(target == RobotTarget.REAL_ROBOT || target == RobotTarget.GAZEBO, jointMap);
```

Comment:
Like I mentioned before, this is a hack, we have a plan to fix this properly. So we'll probably reject this PR but I'm going to leave it open right now because we need to port over your other commits for the F/T sensors.

Reply:
👍 
Just getting this on your radar so you are able to reproduce.